lib/common/diagramvisitor.js

```diff
@@ -161,6 +163,7 @@ class DiagramVisitor {
         classDeclaration.getOwnProperties().forEach((property) => {
             property.accept(this, parameters);
         });
+        this.handleDecorators(classDeclaration, parameters);
```

Comment:
How about decorators for the whole model?

Reply:
How would that work? The decorators target elements in the model and not the model itself.

Reply:
We actually have made that change recently.
https://github.com/accordproject/concerto/pull/877

Reply:
Thanks for bringing this to my notice.